library/threading.c

```diff
@@ -17,7 +17,7 @@
 
 #if defined(MBEDTLS_THREADING_C)
 
-#include "mbedtls/threading.h"
+#include "threading_internal.h"
```

Comment:
Question: in an LTS release, is it OK to introduce a new file that the library uses? Would it be a little bit safer to only introduce the new file but not use is in the library?